src/lib/vpn/main_dialog.rb

```diff
@@ -104,7 +104,7 @@ def fix_mss_help_handler
                 "it is possible that the affected hosts prevent automatic MTU (maximum transmission " +
                 "unit) discovery due to incorrect firewall configuration.\n" +
                 "Reducing TCP-MSS will correct the situation; however, the available bandwidth will be " +
-                "reduced by about 10%."))
+                "reduced by about 12%."))
```

Comment:
If translations already exist, I think it is not worth breaking them, because 12 **is** about 10.